docs/docs/backup_howto.md

```diff
@@ -0,0 +1,205 @@
+# Backup strategy guide
+
+This guide explains how to design and implement a backup strategy in Xen Orchestra.  
+Instead of a simple list of questions and answers, it walks you through **key decisions** and **best practices** before, during, and after setting up backups.
+
+---
+
+## Glossary of backup types & features
+
+- **Full backup** – Copies the entire VM each time, regardless of previous backups.  
+- **Delta backup** – Saves only the changes since the last backup, reducing storage needs.  
+- **Incremental replication** – Formerly called Continuous Replication (CR). Keeps a near-real-time copy of a VM on another host or pool.  
+- **Full replication** – Formerly called Disaster Recovery (DR). Creates a complete replica of a VM on another host or pool.  
+- **Mirror backup** – Keeps an exact mirror of the VM backup set, with no retention—only the latest copy is kept.  
+- **Sequence** – Groups multiple backups or replications in a specific order to be run as a single job.  
+- **Long-term retention** – Keeps backups over extended periods (weeks, months, or years) for compliance or archival purposes.  
```

Comment:
It should be noted somewhere that this should not be mixed with schedule retention otherwise you will have backup duplication.